lib/zendesk_apps_tools/server.rb

```diff
@@ -4,7 +4,6 @@
 module ZendeskAppsTools
   class Server < Sinatra::Base
     set :protection, :except => :frame_options
-    set :public_folder, proc { "#{settings.root}/assets" }
```

Comment:
Wouldn't keeping this and not having your

```ruby
get "/:file"
  ...
end
```

be equivalent?

It's really the `options` block you need, isn't it?